.github/scripts/build-msvc.ps1

```diff
@@ -18,7 +20,13 @@ Expand-Archive $Env:TEMP\SDL2-devel.zip -DestinationPath $Env:TEMP
 
 # build
 cmake -G Ninja -DCMAKE_BUILD_TYPE=RelWithDebInfo -DBUILD_TESTS=ON "-DSDL2_ROOT_DIR=$($Env:TEMP)\SDL2-$sdl2_version" -B build
+if ($LASTEXITCODE -ne 0) {
```

Comment:
Since last week, SDL has https://github.com/libsdl-org/setup-sdl to set-up SDL2/3 in github actions.
It's still experimental, but works very well for cmake projects.

Reply:
cool, adding an issue to look at using it 👍